recipes/nnetsauce/meta.yaml

```diff
@@ -0,0 +1,50 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+{% set name = "nnetsauce" %}
+{% set version = "0.8.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/b2/89/8fa407c95cf2f89c7b79b3ad43c5517382b1628861d6e640376f3d9c53b0/{{ name|lower }}-{{ version }}-cp38-cp38-macosx_10_15_x86_64.whl
+  sha256: 0f82123ffe1b4678f21f2efa79023b254355cd8cb8eb13cef75d7badca54f0eb
+  #url: https://github.com/Techtonique/{{ name|lower }}/tree/master/dist/{{ name|lower }}-{{ version }}.tar.gz
+  #sha256: ea5bf255c06f15062130621154873e9f69eab8f97ee9e3db456906680eb94b0e
+  # RuntimeError: SHA256 mismatch: '544d9a5fff4b4a475a3b51540de4d97070a16684538c9409bc383c61057b13bc
+
+build:
+  number: 0  
+  skip: True  # [py<35]
```

Comment:
you need to specify a build script, either by using the standard:

`script: "{{ PYTHON }} -m pip install . -vv"`
https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L35

or if you need something more complicated, including a file `build.sh` (for mac and linux) ` and `bld.bat` (for windows) in this directory.